pkg/limits/frontend/frontend.go

```diff
@@ -32,37 +29,15 @@ const (
 	ReasonExceedsRateLimit = "exceeds_rate_limit"
 )
 
-type metrics struct {
-	activeStreams          *prometheus.GaugeVec
```

Comment:
These metrics no longer make sense as the frontend is no aggregating global counts.